crates/ide-diagnostics/src/handlers/unlinked_file.rs

```diff
@@ -28,39 +28,40 @@ pub(crate) fn unlinked_file(
     file_id: FileId,
 ) {
     let mut range = TextRange::up_to(ctx.sema.db.line_index(file_id).len());
-    let fixes = fixes(ctx, file_id, range);
-    // FIXME: This is a hack for the vscode extension to notice whether there is an autofix or not before having to resolve diagnostics.
```

Comment:
Fwiw this fixme isn't quite applicable anymore currently. The unlinked file handling the vscode client is completely disabled at the moment. (the vscode extension used to scan the content of the message here to do conditional work which is what this FIXME is referring to)